pyproject.toml

```diff
@@ -19,6 +20,7 @@ goose-plugins = "goose_plugins:module_name"
 
 [project.entry-points."goose.toolkit"]
 artify = "goose_plugins.toolkits.artify:VincentVanCode"
+data = "goose_plugins.toolkits.data:Data"
```

Comment:
I have but one comment. Can we call it a bit less ambiguous? Something like `data-processor` ?
Also can you add it's description to the docs in `docs/plugins/available-toolkits.md`